package.json

```diff
@@ -11,8 +11,7 @@
     "test-integration": "sh scripts/tests/testIntegration.sh",
     "test-unit": "sh scripts/tests/testUnit.sh",
     "test": "yarn lint && yarn test-unit && yarn test-integration",
-    "tdd:watch": "sh scripts/tests/tdd.sh",
```

Comment:
Can we not remove this line? I believe this is not related to swagger.

Reply:
in this, just trailing comma is removed, not the whole line, getting error because of trailing comma